spec/shr_medication.txt

```diff
@@ -64,50 +64,59 @@ Description:	"The dosage of the medication. The data structure is flexible and c
 		Value:			Ratio
 
 
+Abstract Element:	MedicationExposure
+Based on:			ActionPerformed
+Description:	"Abstract class representing medication exposure, independent of whether reported in a medication statement or medication administration. Does not include medication requests (prescriptions)."
+1..1			Medication
+0..1			Dosage
+0..0			Method // because of Dosage.Method
+
+
 EntryElement:	MedicationStatement
-Based on:		ActionPerformed
+Based on:		MedicationExposure
 Concept:		SCT#416608005 "Drug therapy (procedure)"  // TODO: investigate whether we need a new code or use a drug concept.
 Description:	"A record of the use of a medication. If the medication is given as part of a course of treatment, the PartOf attribute should refer to the CourseOfTreatment."
 				Asserter value is type PersonOrOrganization
 1..1			CreationDate // us-core required
-1..1			Medication
-0..1			Dosage
 0..1			Category should be from http://hl7.org/fhir/ValueSet/medication-statement-category
-				PerformanceStatus from http://hl7.org/fhir/ValueSet/medication-statement-status
-0..0			Method // because of Dosage.Method				
-0..1			Reason // DSTU 2 change (upper card imposed by DSTU 2)
+				Status from http://hl7.org/fhir/ValueSet/medication-statement-status		
+0..1			Reason // Upper cardinality imposed by DSTU 2
+0..0			Participation
+0..0			Location
+				RelatedRequest value is type ref(MedicationRequested) // or CarePlan or ProcedureRequest or ReferralRequest
 
 EntryElement:	MedicationAdministered
-Based on:		ActionPerformed
-Concept:		TBD
+Based on:		MedicationExposure
```

Comment:
Maybe rethink the name?  Exposure gets confusing with an environment determinant of health.  For example, I can be exposed to a substance without having it intended to be administered.